### ISSUE TITLE
After update virtualenv_support/pip to 9.0.1 test pypy3 started to fail

### ISSUE THREAD

**Issue by AlexeySanko** [pypa/virtualenv#978]
Pip 9.0.1 raises error: 
```
Collecting setuptools
Collecting pip
/home/travis/build/pypa/virtualenv/virtualenv_support/pip-9.0.1-py2.py3-none-any.whl/pip/_vendor/pkg_resources/__init__.py:83: UserWarning: Support for Python 3.0-3.2 has been dropped. Future versions will fail here.
pip requires Python '>=2.6,!=3.0.*,!=3.1.*,!=3.2.*' but the running Python is 3.2.5
```
[run example](https://travis-ci.org/pypa/virtualenv/jobs/174937963)

**Comment by yan12125** [comment]
Travis CI should be updated to PyPy3 5.5.0
https://github.com/travis-ci/travis-ci/issues/6727

**Comment by yan12125** [comment]
This can be closed since #982
